### PR TITLE
Lb4 body url params

### PR DESCRIPTION
**Pull request by pmcclory-contrast**
~~There's some linting errors around using `any` that need to be fixed or ignored.~~

### CODE REVIEW

File: loopback4/src/vulnerabilities/controllerFactory.ts [hunk 1]
```diff
@@ -49,6 +52,83 @@ function staticVulnerabilityControllerFactory(
...
+
+function formatBodySpec() : RequestBodyObject {
+  const requestBodySpec : RequestBodyObject = {
+    description: 'user input',
+    required: false,
+    content: {
+      'application/json': {
+        schema: {
+          type: 'object',
+	  required: ['input'],
```
Comment (pmcclory-contrast) [inline loopback4/src/vulnerabilities/controllerFactory.ts:87]: yuck - prettier doesn't seem to be running for me - will fix.

File: loopback4/src/vulnerabilities/controllerFactory.ts [hunk 2]
```diff
@@ -49,6 +51,89 @@ function staticVulnerabilityControllerFactory(
...
+  input: string,
+  params: Param[],
+): ParameterObject[] {
+  if (input === 'body') {
+    return [];
+  }
+
+  // mapping of test-bench-utils parameter types
+  // to lb4 parameter types
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (pmcclory-contrast) [inline loopback4/src/vulnerabilities/controllerFactory.ts:64]: We were already allowing `any` in at least one other spot - I don't really care to enumerate all of the types myself.

File: loopback4/src/vulnerabilities/controllerFactory.ts [hunk 3]
```diff
@@ -106,8 +183,10 @@ function vulnerabilityControllerFactory(
         async safe(
           @inject(RestBindings.Http.REQUEST) req: Request,
           @inject(RestBindings.Http.RESPONSE) res: Response,
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (tough-griff) [inline loopback4/src/vulnerabilities/controllerFactory.ts:186]:
```suggestion
```

File: loopback4/src/vulnerabilities/controllerFactory.ts [hunk 4]
```diff
@@ -106,8 +183,10 @@ function vulnerabilityControllerFactory(
...
+          ...args: any[]
```
Comment (tough-griff) [inline loopback4/src/vulnerabilities/controllerFactory.ts:187]: aha!!

**Review by tough-griff — CHANGES_REQUESTED** [review]
Throw that eslint-disable up to the top and we're golden.

Gonna load it up and test, then should be ready to approve.

**Pull request merged**